back/app/controllers/web_api/v1/areas_controller.rb

```diff
@@ -13,13 +13,19 @@ def index
     @areas = paginate @areas
 
     include_static_pages = params[:include]&.split(',')&.include?('static_pages')
+    user_followers = current_user&.follows
+      &.where(followable_type: 'Area')
+      &.group_by do |follower|
+        [follower.followable_id, follower.followable_type]
+      end
+    user_followers ||= {}
```

Comment:
This is a pattern that is repeated in many places: current_basket, current_reaction, user_attendance etc. I was wondering if it could be worth introducing helper methods for this for controllers and serializers, but felt like it was too high effort to squeeze in.

How would a separate endpoint work with e.g. ideas#index? Would the frontend fetch all ideas and user votes separately, and then merge them together with application code?